gitbootcamp.rst

```diff
@@ -157,9 +157,29 @@ You may specify more than one branch for deletion.
 Renaming Branch
 ---------------
 
-The CPython repository's default branch was renamed from ``master`` to ``main``
-after the Python 3.10b1 release. If you had cloned the repository before this
-change, you can rename your local branch as follows::
+The CPython repository's default branch was renamed from ``master`` to
+``main`` after the Python 3.10b1 release. If you have a fork on GitHub (as
```

Comment:
I was thinking that this portion

> If you have a fork on GitHub (as .....
> ....
> ... by following these GitHub instruction

can be on its own paragraph.

The first sentence statin the fact that the default branch was renamed could be a standalone sentence. I think it would read better that way.

What do you think?

Reply:
Sure. Done.